include/seqan/align/dp_algorithm_impl.h

```diff
@@ -326,100 +370,38 @@ _computeTrack(TDPScout & scout,
         _goNextCell(dpScoreMatrixNavigator, TColumnDescriptor(), InnerCell());
         _goNextCell(dpTraceMatrixNavigator, TColumnDescriptor(), InnerCell());
         // Compute the inner cell.
-        _computeCell(scout, dpTraceMatrixNavigator, value(dpScoreMatrixNavigator),
-                     previousCellDiagonal(dpScoreMatrixNavigator), previousCellHorizontal(dpScoreMatrixNavigator),
-                     previousCellVertical(dpScoreMatrixNavigator), seqHValue,
-                     sequenceEntryForScore(scoringScheme, container(iter), position(iter)), scoringScheme,
-                     TColumnDescriptor(), InnerCell(), TDPProfile());
-//        std::cerr << _scoreOfCell(value(dpScoreMatrixNavigator)) << " \t";
-//        std::cerr << _scoreOfCell(value(dpScoreMatrixNavigator)) << "(" << _horizontalScoreOfCell(value(dpScoreMatrixNavigator)) << "," << _verticalScoreOfCell(value(dpScoreMatrixNavigator)) << ")" << " \t";
+        // If we have variable length simd, we need to check if we reached the end of one of the sequences.
+        // For all other cases, the function returns always false.
+        if (SEQAN_UNLIKELY(_reachedVerticalEndPoint(scout, iter)))
```

Comment:
Actually, not, but it shouldn't because the _reachedVerticalEndPoint() returns always false, except for one special ScoutType. So I hope the compiler will optimise it away.
But sure, I will benchmark it on different platforms.